apps/meeting/src/utils/api.ts

```diff
@@ -11,23 +11,26 @@ interface MeetingResponse {
     Meeting: any;
   };
 }
+interface GetAttendeeResponse {
+  name: any;
```

Comment:
Just wonder if we are improve the code, can we change this `any` to some specific type like `string`?

Reply:
Checked the API definition. The type of return data is `string`, so updated `any` to `string`.